### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by npinsker**
rewrite README to better answer the question "should i download this?" by giving an overview of features

- move user guide to its own .md file (i didn't delete any existing text, it's all in `user-guide.md`)
- listed out everything I know about
- please help me fill in what i don't know about

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,76 +2,32 @@
...
+* Quickly search and filter on puzzle names, rounds, and tags
+* Remote-friendly! Real-time updates on how recently each puzzle's sheet was active, and who's currently working on it
 
-When a puzzle is added, a Google Sheet is automatically created for it, linked from the Sheet icon. The puzzle sheet is a clone of a template sheet, which may have some useful formulas pre-populated. Ownership of the puzzle sheets is transferred to the owner of the the template sheet - search `pendingowner:me` to see and accept the transfers.
+### Integrations
 
-### Tools and References
+* Creates a Google Sheet for new puzzles automatically
+* Creates and manages Discord channels for each puzzle
+* TODO: does it do more than just channel management?
```
Comment (akirabaruah) [inline README.md:26]: We technically have a Discord bot built-in, but that hasn't been really used / playtested much, so probably not worth advertising here.

File: README.md [hunk 2]
```diff
@@ -2,76 +2,32 @@
...
+* Chrome extension (TODO: what does this do)
```
Comment (akirabaruah) [inline README.md:27]: @maximized Wrote the initial implementation. IIUC, it basically provides a pop-up version of the "add puzzle" form, but I could be missing other functionality.

File: README.md [hunk 3]
```diff
@@ -2,76 +2,32 @@
...
+* TODO: check with others if there's anything else
```
Comment (akirabaruah) [inline README.md:28]: Should we include Google Sheet activity tracking here? @rawxfish

File: README.md [hunk 4]
```diff
@@ -2,76 +2,32 @@
...
+For help **setting Cardboard up for a hunt**, see [here](new-hunt-setup.md).
 
-Meta puzzles may also be assigned to other metas (which might be meta-metas).
+For information about **development**, see the [dev guide](dev-guide.md).
 
-In addition to meta tags for puzzles, users can also tag puzzles as high or low priority, logic or word puzzles, or create new tags.
+### Features
 
-### Google Sheets integration
+* Seamlessly supports (almost) any puzzle hunt structure
```
Comment (akirabaruah) [inline README.md:17]: (no action) "seamless" seems a bit confident lol, but I guess we have used it for several medium to large sized puzzle hunts :D

Comment (akirabaruah) [reply README.md:17]: Actually, on that note, would it make sense to list example puzzle hunts that we recommend using this tool for? e.g. MITMH, Puzzle Boat, etc.?

**Review by akirabaruah — APPROVED** [review]
Thanks for the cleanup! LGTM

File: README.md (hunk 2 above)
Comment (akirabaruah) [reply README.md:27]: Oh to clarify, it grabs the current URL and page title to auto-populate the form when adding a new puzzle.

File: README.md [hunk 5]
```diff
@@ -2,76 +2,32 @@
...
+* Create and assign tags to puzzles
+* Quickly search and filter on puzzle names, rounds, and tags
+* Remote-friendly! Real-time updates on how recently each puzzle's sheet was active, and who's currently working on it
 
-When a puzzle is added, a Google Sheet is automatically created for it, linked from the Sheet icon. The puzzle sheet is a clone of a template sheet, which may have some useful formulas pre-populated. Ownership of the puzzle sheets is transferred to the owner of the the template sheet - search `pendingowner:me` to see and accept the transfers.
+### Integrations
 
-### Tools and References
+* Creates a Google Sheet for new puzzles automatically
```
Comment (rgossiaux) [inline README.md:24]: The Sheets integration has a couple other fancy features beyond creation, like renaming sheets when puzzles are solved and inserting all feeder answers into meta sheets automatically (@rawxfish am I missing anything?)

File: user-guide.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+Any user can add a new puzzle by using the blue "Add New Puzzle" button in the top right.
+
+<img src='https://user-images.githubusercontent.com/544734/71759777-3cdc8d00-2e80-11ea-9d49-48de77370976.png' width='300'>
+
+You must specify a name but the URL is optional. If you know the puzzle is a meta puzzle, you can check the "Meta?" checkbox. In case you made a mistake or realize a puzzle is a meta later on, you can always go back and edit a puzzle by pressing the pencil icon next to the puzzle name.
+
+In case you need to delete a puzzle (perhaps you accidentally created a duplicate), you can do so using the trash icon next to a puzzle name.
+
+When a puzzle is created, a Google Spreadsheet are automatically created for the puzzle. These are linked from the Sheets icon.
```
Comment (rgossiaux) [inline user-guide.md:43]: is* automatically

**Pull request merged**